packages/markdown/remark/src/remark-content-rel-image-error.ts

```diff
@@ -0,0 +1,47 @@
+import type { Image } from 'mdast';
+import { visit } from 'unist-util-visit';
+import type { VFile } from 'vfile';
+
+/**
+ * `src/content/` does not support relative image paths.
+ * This plugin throws an error if any are found
+ */
+export default function remarkContentRelImageError() {
+	return (tree: any, vfile: VFile) => {
+		if (!vfile.path.includes('content/')) return;
```

Comment:
This could have false positives—would ``vfile.path.startsWith(`${vfile.cwd}/src/content/`)`` be more precise?

Reply:
Ah, stubbed that out and thought I fixed it 🤦‍♂️ We'd probably pull in the `srcDir` from the user config and join up here. Will update!